Code/Tools/AssetProcessor/native/resourcecompiler/RCBuilder.cpp

```diff
@@ -684,9 +684,15 @@ namespace AssetProcessor
                     ++retryCount;
                     ProcessLegacyRCJob(request, rcParam, assetRecognizer->m_productAssetType, jobCancelListener, response);
 
+                    // If a lost connection occured, prepare for a retry using an exponential backoff policy
+                    if ((response.m_resultCode == AssetBuilderSDK::ProcessJobResult_NetworkIssue) && (retryCount <= AssetProcessor::RetriesForJobLostConnection))
+                    {
+                        const int delay = aznumeric_caster(pow(2, retryCount-1));
```

Comment:
`pow` is function that performs a floating point calculation.
Instead `1 << retryCount - 1` should be used

Reply:
thats a good point, but this is a cherry pick from development so i'm not sure if that should be done in stabilization

Reply:
I must ignored the original PR then.

Reply:
I'll make the update